doc/source/whatsnew/v0.23.0.txt

```diff
@@ -1111,6 +1111,7 @@ Indexing
 - Bug in :meth:`DataFrame.first_valid_index` and :meth:`DataFrame.last_valid_index` in presence of entire rows of NaNs in the middle of values (:issue:`20499`).
 - Bug in :class:`IntervalIndex` where some indexing operations were not supported for overlapping or non-monotonic ``uint64`` data (:issue:`20636`)
 - Bug in ``Series.is_unique`` where extraneous output in stderr is shown if Series contains objects with ``__ne__`` defined (:issue:`20661`)
+- Bug in ``.loc`` where assigned value to ``DataFrame`` is unexpectedly listed (:issue:`19474`)
```

Comment:
Bug in ``.loc`` assignment with a single-element list-like incorrectly assigns as a list.